easybuild/easyconfigs/y/yank/yank-0.25.2-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,108 @@
+    ('AmberTools', '20', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
+    ('six', '1.15.0', {
+        'checksums': ['30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259'],
+    }),
+    ('lxml', '4.6.1', {
+        'checksums': ['c152b2e93b639d1f36ec5a8ca24cde4a8eefb2b6b83668fcd8e83a67badcb367'],
+    }),
+    ('numpydoc', '1.1.0', {
+        'checksums': ['c36fd6cb7ffdc9b4e165a43f67bf6271a7b024d0bb6b00ac468c9e2bfc76448e'],
+    }),
+    ('openmoltools', '0.8.7', {
+        'modulename': False,
+        'source_tmpl': '0.8.7.tar.gz',
+        'source_urls': ['https://github.com/choderalab/openmoltools/archive'],
+        'checksums': ['dae37dc1136cbc6175b1935e2b4352cc7cc9c9e4d441968178c9e90ec674a1df'],
+    }),
+    ('mpiplus', 'v0.0.1', {
+        'source_tmpl': 'v0.0.1.tar.gz',
+        'source_urls': ['https://github.com/choderalab/mpiplus/archive'],
+        'checksums': ['a3d50157a274decab87589ad78b9e41e9f3d34eb27a7d50cc6e7ef34def98b44'],
+    }),
+    ('clusterutils', '0.3.1', {
+        'source_tmpl': '0.3.1.tar.gz',
+        'source_urls': ['https://github.com/choderalab/clusterutils/archive'],
+        'checksums': ['2c841215d1a93af5c61fac2ff52cdc8a286f05af219bca84a76a7a7380f3ee96'],
+    }),
+    ('sphinxcontrib-bibtex', '1.0.0', {
+        'modulename': 'sphinxcontrib.bibtex',
+        'checksums': ['629612b001f86784669d65e662377a482052decfd9a0a17c46860878eef7b9e0'],
+    }),
+    ('Cerberus', '1.3.2', {
+        'checksums': ['302e6694f206dd85cb63f13fd5025b31ab6d38c99c50c6d769f8fa0b0f299589'],
+    }),
+    ('oset', '0.1.3', {
+        'checksums': ['4c1fd7dec96eeff9d3260995a8e37f9f415d0bdb79975f57824e68716ac8f904'],
+    }),
+    ('latexcodec', '2.0.1', {
+        'checksums': ['2aa2551c373261cefe2ad3a8953a6d6533e68238d180eb4bb91d7964adb3fe9a'],
+    }),
+    ('pybtex', '0.23.0', {
+        'checksums': ['b92be18ccd5e9a37895949dcf359a1f6890246b73646dddf1129178ee12e4bef'],
+    }),
+    ('pybtex-docutils', '0.2.2', {
+        'checksums': ['ea90935da188a0f4de2fe6b32930e185c33a0e306154322ccc12e519ebb5fa7d'],
+    }),
+    ('pdbfixer', 'v1.6', {
+        'source_tmpl': 'v1.6.tar.gz',
+        'source_urls': ['https://github.com/openmm/pdbfixer/archive'],
+        'checksums': ['8dc928d1a57dded981835a6175760f3407c52664b6204053636f7754787a655e'],
+    }),
+    (name, version, {
+        'modulename': False,
+        'source_tmpl': '0.25.2.tar.gz',
+        'source_urls': ['https://github.com/choderalab/yank/archive'],
+        'checksums': ['6a1c91f8afb1d99eac67ef4a22a137f2fd755275f1a614d1ec267cb36b8960d5'],
+    }),
+]
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/yank'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    'mpirun -np 1 yank selftest',
```

Comment:
add a comment above to explain why `mpirun` is used:

```python
# run sanity check commands why mpirun, because they trigger an MPI_Init;
# the impi version in intel/2020a requires that everything that triggers MPI_Init is run via mpirun...